### PR TITLE
Feat/UI/account

### PR DESCRIPTION
**Pull request by sammy-code98**
### **Pull Request Summary: User Account Enhancements and UI Improvements**  

This PR introduces new features and UI updates to improve the user account experience and overall navigation flow. Below are the key highlights:  

#### **New Features:**  
- **User Dropdown Modal:**
  - Added routes and pages accessible via the user dropdown menu for a smoother navigation experience.  

- **Logout Confirmation Modal:**
  - Implemented a modal to confirm user logout actions, enhancing security and preventing accidental logouts.  

- **Account Page:**
  - Created a dedicated account page for users to manage their personal information and settings.  

#### **UI Enhancements:**  
- **Add Shortlet Form:**
  - Updated icons in the "Add Shortlet" form to improve visual clarity and user interaction.  

These updates enhance both the user experience and account management functionality, providing a more intuitive and seamless interface for end-users.

### CODE REVIEW

File: host/src/App.tsx [hunk 1]
```diff
@@ -19,6 +19,8 @@ import Payments from "@/pages/Dashboard/Payments";
 import Settings from "@/pages/Dashboard/Settings/index";
 import ShortletDetails from "@/pages/Dashboard/Shortlet/shortletDetails";
 import EditShortlet from "@/pages/Dashboard/Shortlet/editShortlet";
+import Account from "./pages/Dashboard/Account";
```
Comment (chimdie) [inline host/src/App.tsx:22]:
```suggestion
import Account from "@/pages/Dashboard/Account";
```

File: host/src/App.tsx [hunk 2]
```diff
@@ -19,6 +19,8 @@ import Payments from "@/pages/Dashboard/Payments";
...
+import Notification from "./pages/Notification";
```
Comment (chimdie) [inline host/src/App.tsx:23]:
```suggestion
import Notification from "@/pages/Notification";
```

File: host/src/components/LogoutModal.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  onClose,
+  onOpenChange,
+}: LogoutModalProps): JSX.Element {
+  return (
+    <Modal size="md" isOpen={isOpen} onOpenChange={onOpenChange}>
+      <ModalContent>
+        <ModalBody>
+          <div className="flex flex-col p-4 space-y-8">
+            <div className="rounded-full bg-gray-100 w-14 h-14 flex justify-center items-center mx-auto">
+              <div className="rounded-full bg-white w-10 h-10 flex justify-center items-center">
```
Comment (chimdie) [inline host/src/components/LogoutModal.tsx:21]:
```suggestion
              <div className="rounded-full bg-white w-10 h-10 flex justify-center items-center aspect-square">
```

File: host/src/components/LogoutModal.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+              </p>
+            </div>
+          </div>
+        </ModalBody>
+        <ModalFooter>
+          <div className="flex justify-between gap-6">
+            <Button
+              onClick={onClose}
+              size="lg"
+              fullWidth={true}
```
Comment (chimdie) [inline host/src/components/LogoutModal.tsx:41]:
```suggestion
              fullWidth
```

File: host/src/components/LogoutModal.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+              className="bg-white shadow-sm border text-shorttiee_primary font-medium"
+            >
+              No, keep me logged in
+            </Button>
+            <Button
+              size="lg"
+              fullWidth={true}
```
Comment (chimdie) [inline host/src/components/LogoutModal.tsx:48]:
```suggestion
              fullWidth
```

File: host/src/types/routes.ts [hunk 6]
```diff
@@ -13,6 +13,8 @@ export const DashboardRoutes = {
   editShortlet: "/shortlet/edit/:id",
   settings: "/settings",
   payment: "/payments",
+  account: "/account",
```
Comment (chimdie) [inline host/src/types/routes.ts:16]:
```suggestion
  account: "/profile",
```

**Review by chimdie — CHANGES_REQUESTED** [review]

**Comment by micaiah-effiong** [comment]
Move this `hook` to a separate file. 
https://github.com/chimdie/shorttiee/blob/a724e9c00dd5e9b06e57f1af01beeed7aae89b6f/host/src/components/ui/sidebar.tsx#L34-L41
It currently affects fast refresh during development
image

**Comment by micaiah-effiong** [comment]
@sammy-code98 side-bar active state is removed on-hover

**Comment by sammy-code98** [comment]
> Move this `hook` to a separate file.
> 
> https://github.com/chimdie/shorttiee/blob/a724e9c00dd5e9b06e57f1af01beeed7aae89b6f/host/src/components/ui/sidebar.tsx#L34-L41
> 
> 
> It currently affects fast refresh during development
> image

@micaiah-effiong  resolved this earlier on , you can review

**Review by chimdie — APPROVED** [review]

**Pull request merged**